src/main/webapp/Components/footer.jsp

```diff
@@ -32,7 +32,7 @@
 
 		<div>
 			<div>
-				<img src="resources/img/cancelletto.jpg" class="logo-footer">
```

Comment:
Per questo componente è meglio usare `${pageContext.request.contextPath}` in modo che prenda sempre il contesto dalla pagina di partenza del server e quindi non dia problemi nell'import delle immagini.